frontend/wallet/package.json

```diff
@@ -13,16 +13,20 @@
   "devDependencies": {
     "bs-platform": "^4.0.18",
     "concurrently": "^4.1.0",
-    "fastpack": "^0.8.4"
+    "fastpack": "^0.8.4",
+    "graphql-faker": "^1.9.0"
   },
   "scripts": {
     "start": "electron ./lib/js/src/App.js",
+    "fake": "graphql-faker --port 8080 -- schema.graphql",
+    "fake-inspector": "graphql-faker --open -- schema.graphql",
     "build": "bsb -make-world",
     "clean": "bsb -clean-world",
     "pack": "fpack --development ./lib/js/src/Index.js",
     "query": "send-introspection-query http://localhost:8080/graphql",
+    "query-fake": "concurrently 'npm run fake' 'sleep 5 && npm run query && echo You can now stop this process'",
     "reformat": "bsrefmt --in-place src/*.re",
-    "dev": "concurrently 'bsb -make-world -w' 'fpack watch --development ./lib/js/src/Index.js' 'electron ./lib/js/src/App.js'",
+    "dev": "concurrently 'npm run fake' 'bsb -make-world -w' 'fpack watch --development ./lib/js/src/Index.js' 'sleep 2 && electron ./lib/js/src/App.js'",
```

Comment:
Do we need the `sleep 2`? Can we instead `(concurrently foo bar) && electron ./lib/js/src/App.js`?

Reply:
I don't think so, because concurrently won't end on its own

Reply:
it's 2 seconds not 2 millis right? Does this only happen once at startup or every change?

Reply:
at startup, 2 seconds